plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/glue/TestingIcebergGlueCatalogModule.java

```diff
@@ -62,18 +63,16 @@ protected void setup(Binder binder)
         newExporter(binder).export(TrinoCatalogFactory.class).withGeneratedName();
         binder.bind(AWSGlueAsyncAdapterProvider.class).toInstance(awsGlueAsyncAdapterProvider);
 
+        Multibinder<RequestHandler2> requestHandlers = newSetBinder(binder, RequestHandler2.class);
+        install(conditionalModule(
+                IcebergGlueCatalogConfig.class,
+                IcebergGlueCatalogConfig::isSkipArchive,
+                config -> requestHandlers.addBinding().toInstance(new SkipArchiveRequestHandler())));
```

Comment:
- `config` is of type `Binder`. rename the parameter
- i don't think one module should add to set binder created by another module; is it guaranteed to work?
  - repeat `newSetBinder(binder, RequestHandler2.class)` inside the conditional module

Reply:
The quoted text says that 2+ modules can call `newSetBinder(binder, X.class)`  and this will end-up being one `Set<X>`.

it does not seem to say, however, that ``newSetBinder(binder, X.class)`` result (the `Multibinder` object) can be shared between modules